stable/aws-for-fluent-bit/values.yaml

```diff
@@ -108,7 +105,6 @@ elasticsearch:
   replaceDots: "On"
 
 # extraOutputs: |
-#   [OUTPUT]
```

Comment:
Why removing this single line comment? Either remove the whole output section or don't touch them. Same for above input and filter section.

Reply:
Yes @PettitWesley, as I explained in my introduction above `and to remove the block configuration title [FILTER] [INPUT] and [OUTPUT] in comments in values file.` The reason is it makes people confused and if you keep two blocks, it won't work...